components/posts/recommandedPosts.js

```diff
@@ -38,11 +38,13 @@ export default function RecommandedPosts({ postData }) {
                           alt={post.authorAltText}
                         />
                       </div>
-                      <span className="font-medium">{post.authorName}</span>
+                      <span className="font-worksans-medium tracking-wide">
+                        {post.authorName}
+                      </span>
                     </div>
 
                     <div
-                      className="text-[1rem] sm:text-[1.1rem] md:text-[1.25rem] xl:text-[1rem] font-bold leading-[1.27rem] md:leading-[1.7rem] xl:leading-5 tracking-[0.06rem] xl:line-clamp-3"
+                      className="text-[1rem] sm:text-[1.1rem] md:text-[1.25rem] xl:text-[1.01rem] leading-[1.27rem] md:leading-[1.7rem] xl:leading-5 tracking-[0.01rem] xl:line-clamp-3 text-[#000000] font-medium font-worksans-medium"
```

Comment:
use default tailwind class for text-size.

Reply:
This default tailwind class does not provide the same leading that I need with this text. i will used in website if any place default tailwind class work.

Reply:
there is class "text-base" can be used in place of text-[1rem]

Reply:
that write but with that provide line-height=1.5rem and i need to **line-height=1.27rem**. so that

Reply:
if you use you own leading-size it will override the leading of default classs. so you can use it freely.